docs/CRATES_IO_RELEASE.md

```
@@ -253,10 +271,20 @@ git commit -m "chore(release): vX.Y.Z-alpha.1"
 7. Verify `HEAD` equals `origin/main`.
 8. Verify the release commit is already reachable from origin/main before
    creating the tag.
+9. Verify every target-milestone issue except the release gate is closed, moved
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 6; Actual: 9; Style: 1/2/3

(MD029, ol-prefix)